File: 3 - Divisible and Non-divisible Sums Difference/sl.js
var differenceOfSums = function (n , m) {
    let num1 = 0;
    let num2 = 0;
    const range = [1, n + 1];
    for (let i = 0 ; i <= n; i++){
        if(i % m != 0){
            num1 += i
        } else {
            num2 += i
        }
    }
    console.log(num1 - num2)
};

differenceOfSums(10,3)

File: Leetcode/1.March-2024/03 - Divisible and Non-divisible Sums Difference/sl.py
def differenceOfSums( n: int, m: int) -> int:

    nums1:list[int] = []
    nums2:list[int] = []
    for i in range(1,n+1):  
        if i % m != 0 :
            nums1.append(i)
        else:
            nums2.append(i)

    return sum(nums1) - sum(nums2)
